README.rst

```diff
@@ -310,6 +310,326 @@ Multitenancy mixins
+
+.. code-block:: python
+
+    INSTALLED_APPS = [
+        # ... other apps ...
+
+        # 'openwisp_users'  <-- comment out or delete this line
+        'myusers'
+    ]
+
+For more information about how to work with django projects and django apps, please refer
+to the `django documentation <https://docs.djangoproject.com/en/dev/intro/tutorial01/>`_.
+
+2. Install ``openwisp-users``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Install (and add to the requirement of your project) openwisp-users::
+
+    pip install openwisp-users
+
+3. Add ``EXTENDED_APPS``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add the following to your ``settings.py``:
+
+.. code-block:: python
+
+    EXTENDED_APPS = ('openwisp_users',)
+
+
+4. Add ``openwisp_utils.staticfiles.DependencyFinder``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add ``openwisp_utils.staticfiles.DependencyFinder`` to
+``STATICFILES_FINDERS`` in your ``settings.py``:
+
+.. code-block:: python
+
+    STATICFILES_FINDERS = [
+        'django.contrib.staticfiles.finders.FileSystemFinder',
+        'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+        'openwisp_utils.staticfiles.DependencyFinder',
+    ]
+
+5. Add ``openwisp_utils.loaders.DependencyLoader``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add ``openwisp_utils.loaders.DependencyLoader`` to ``TEMPLATES`` in your ``settings.py``:
+
+.. code-block:: python
+
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'OPTIONS': {
+                'loaders': [
+                    'django.template.loaders.filesystem.Loader',
+                    'django.template.loaders.app_directories.Loader',
+                    'openwisp_utils.loaders.DependencyLoader',
+                ],
+                'context_processors': [
+                    'django.template.context_processors.debug',
+                    'django.template.context_processors.request',
+                    'django.contrib.auth.context_processors.auth',
+                    'django.contrib.messages.context_processors.messages',
+                ],
+            },
+        }
+    ]
+
+6. Inherit the AppConfig class
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please refer to the following files in the sample app of the test project:
+
+- `openwisp_users/__init__.py <https://github.com/openwisp/openwisp-users/tree/master/tests/openwisp2/openwisp_users/__init__.py>`_
+- `openwisp_users/apps.py <https://github.com/openwisp/openwisp-users/tree/master/tests/openwisp2/openwisp_users/apps.py>`_
+
+You have to replicate and adapt that code in your project.
+
+
+For more information regarding the concept of ``AppConfig`` please refer to
+the `"Applications" section in the django documentation <https://docs.djangoproject.com/en/dev/ref/applications/>`_.
+
+7. Create your custom models
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the purpose of showing an example, we added a simple "fiscal_number" field in User model to the
```

Comment:
This is great!

I am just not convinced of "fiscal number", is it a term you have heard before? In Italy it's called "Codice fiscale" but I don't think other nationalities will get it.

What about "Social Security Number"? Spain has "Seguridad Social" which is very close so I suspect this term may be understood by a broader audience.

We could then write something like:

```
This is the classic example of field that varies a lot from country to country, every country has a different name and different validation rules, so that's why these are not included by default in OpenWISP and is left up to the user to implement them.
```

Some countries even have libraries or services to validate these IDs, so we may point out that custom validation and verification systems can be implemented as well.

In the example images, please use a [sample SSN like the ones shown in google images](https://www.google.com/search?q=social+security+number+example&client=ubuntu&hs=Une&sxsrf=ALeKk00-fPXIb5gpuxbv4CwiuZ2NmmQn_w:1591380396346&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjUuOf5oevpAhU2IrkGHSLjD_UQ_AUoAXoECA4QAw&biw=1920&bih=863#imgrc=RhJ7I8-HJbnWDM), to make the example more realistic.

Reply:
Okay, I'll do that.
How about a field name like "Employee Number" or "College Enrollment Number"; conveys the message that a custom field is added and I think everyone will get that.

Reply:
> In the example images, please use a sample SSN like the ones shown in google images, to make the example more realistic.

I didn't get that, do you mean, I should add an image of this in the README?

Reply:
> Okay, I'll do that.
> 
> > In the example images, please use a sample SSN like the ones shown in google images, to make the example more realistic.
> 
> I didn't get that, should I add a link to an image of this in the README?

I meant to use a realistic SSN and take the screenshots using the realistic SSN (instead of 1234)

> How about a field name like "Employee Number" or "College Enrollment Number"; conveys the message that a custom field is added and I think everyone will get that.

I like SSN because it's one of those things that people interested in OpenWISP ask.